images.yml

```diff
@@ -11,13 +11,13 @@
   patches: alpine/patches/26+
 
 - tags: [26.0, 26]
```

Comment:
The tags should be 26.1 no?

Reply:
not sure. 26.1 is release number. I guess people will expect 26.1 to never change.

Reply:
Hum. Not sure either, but I think we can't be wrong with `26.1-rc1`.

Reply:
I have added 26.1-rc1. The problem is that all builds referencing 26.0 will break now. I'm not sure that's a big problem though.

Reply:
The build should reference `26` for now, then `26.1` once it's out.